ComputeDataDrivenCrossSection.py

```diff
@@ -161,8 +161,13 @@
         frac = fFD
         uncFrac = fFDUnc
 
-    crossSec, crossSecUnc = ComputeCrossSection(rawYield, rawYieldUnc, frac, uncFrac, effAcc,
-                                                ptMax - ptMin, 1., sigmaMB, nEv, 1.)
+    if args.FD:
+        crossSec, crossSecUnc = ComputeCrossSection(rawYield, rawYieldUnc, frac, uncFrac, effAcc,
+                                                    ptMax - ptMin, 1., sigmaMB, nEv, 1., 'corr')
+    else:
+        crossSec, crossSecUnc = ComputeCrossSection(rawYield, rawYieldUnc, frac, uncFrac, effAcc,
+                                                    ptMax - ptMin, 1., sigmaMB, nEv, 1., 'anticorr') # TODO: check this
```

Comment:
Not sure that they are anti-correlated for the prompt measurement, but we can check this better in future

Reply:
They should be anticorrelated because the prompt and FD components from the chi2 minimisation are anticorrelated, but indeed we can recheck the first time that we compute a data-driven cross section for prompt

Reply:
Yes, but for the prompt case we are estimating the prompt fraction in a region "far away" from the one where we extract the raw yield, so not sure if the fully anti-correlated hypothesis still holds.

Reply:
The anticorrelation should hold in any case, because it is between N_FD and N_prompt which are used to compute each prompt or FD fraction, regardless the efficiencies (and when you compute the fractions, the uncertainties on N_FD and N_prompt are those that matters)